contents/handbook/company/security.md

```diff
@@ -95,10 +95,12 @@ To provide a great customer experience, PostHog employees may occasionally need
 1. **Only impersonate when there’s a clear, demonstrable benefit for the customer.**  
    For example, to investigate an incident, resolve a support issue, or review a customer’s setup to give recommendations on how to use PostHog more successfully.
 
-2. **Do not make any changes to a customer’s setup without explicit consent.** Exceptions to this are cases where we are reacting to incidents or bad configurations that are negatively impacting PostHog services in order to protect ourselves _and_ the customer.
+2. **Do not make any changes to a customer’s setup without explicit consent.**
+   Exceptions to this are cases where we are reacting to incidents or bad configurations that are negatively impacting PostHog services in order to protect ourselves _and_ the customer.
 
 3. **Ask for permission whenever possible.**  
    While this isn’t always feasible, such as during an active incident, it’s best practice to inform the customer before accessing their account.
+   When a customer raises a support ticket, we take this as consent to be able to impersonate their account and investigate based on the contents of the ticket. Customers will not be actively asked for permission by our support engineers when they are investigating a ticket, and the customer should inform us in the ticket if they explicitly do not wish for our support engineers to access their account.
```

Comment:
We currently run Zendesk in an open setup, which means the ticket could have been created with a spoofed email address. We'd need to operate in a closed setup to be able to trust that it was actually the customer that raised the ticket.

https://support.zendesk.com/hc/en-us/articles/4408887573274-Understanding-options-for-end-user-access-and-sign-in#topic_dkn_nfk_v3